sumpy/p2e.py

```diff
@@ -43,12 +43,14 @@
 
 # {{{ P2E base class
 
-class P2EBase(KernelComputation, KernelCacheWrapper):
+class P2EBase(KernelCacheMixin, KernelComputation):
     """Common input processing for kernel computations.
 
     .. automethod:: __init__
     """
 
+    default_name = "p2p"
```

Comment:
```suggestion
    default_name = "p2e"
```

Reply:
Nice catch! Should be fixed in latest version.